src/sentry/static/sentry/app/views/settings/components/dataScrubbing/modals/form/sourceField.tsx

```diff
@@ -283,6 +268,52 @@ class SourceField extends React.Component<Props, State> {
     }
 
     return fieldValues;
+  }
+
+  checkPossiblyRegExMatchExpression(value: string) {
+    const {isRegExMatchesSelected} = this.props;
+    const {help} = this.state;
+
+    if (isRegExMatchesSelected) {
+      if (help) {
+        this.setState({help: ''});
+      }
+      return;
+    }
+
+    const isPossiblyARegularExpression = RegExp('^/.*/g?$').test(value);
```

Comment:
Maybe something shorter?

```suggestion
    const isMaybeRegExp = RegExp('^/.*/g?$').test(value);
```